astropy_helpers/commands/build_ext.py

```diff
@@ -1,329 +1,180 @@
-                    cython_version = None
-
-                if (cython_version is not None and
-                        cython_version != self.uses_cython):
-                    self.force_rebuild = True
-                    # Update the used cython version
-                    self.uses_cython = cython_version
-
-            # Regardless of the value of the '--force' option, force a rebuild
-            # if the debug flag changed from the last build
-            if self.force_rebuild:
-                self.force = True
-
-        def run(self):
-            # For extensions that require 'numpy' in their include dirs,
-            # replace 'numpy' with the actual paths
-            np_include = None
-            for extension in self.extensions:
-                if 'numpy' in extension.include_dirs:
-                    if np_include is None:
-                        np_include = get_numpy_include_path()
-                    idx = extension.include_dirs.index('numpy')
-                    extension.include_dirs.insert(idx, np_include)
-                    extension.include_dirs.remove('numpy')
-
-                self._check_cython_sources(extension)
-
-            super(build_ext, self).run()
-
-            # Update cython_version.py if building with Cython
-            try:
-                cython_version = get_pkg_version_module(
-                    packagename, fromlist=['cython_version'])[0]
-            except (AttributeError, ImportError):
-                cython_version = 'unknown'
-            if self.uses_cython and self.uses_cython != cython_version:
-                build_py = self.get_finalized_command('build_py')
-                package_dir = build_py.get_package_dir(packagename)
-                cython_py = os.path.join(package_dir, 'cython_version.py')
-                with open(cython_py, 'w') as f:
-                    f.write('# Generated file; do not modify\n')
-                    f.write('cython_version = {0!r}\n'.format(self.uses_cython))
-
-                if os.path.isdir(self.build_lib):
-                    # The build/lib directory may not exist if the build_py
-                    # command was not previously run, which may sometimes be
-                    # the case
-                    self.copy_file(cython_py,
-                                   os.path.join(self.build_lib, cython_py),
-                                   preserve_mode=False)
-
-                invalidate_caches()
-
-        def _check_cython_sources(self, extension):
-            """
-            Where relevant, make sure that the .c files associated with .pyx
-            modules are present (if building without Cython installed).
-            """
-
-            # Determine the compiler we'll be using
-            if self.compiler is None:
-                compiler = get_default_compiler()
-            else:
-                compiler = self.compiler
+    uses_cython = False
+    force_rebuild = False
```

Comment:
I imagine at this point the answer is "it's a breaking change"... but isn't there alse a ``--force`` option?  Is that different from ``--force-rebuild``?  If not, perhaps a deprecation is called for?

Reply:
In fact ``force_rebuild`` is mostly an internal flag in this class that isn't actually needed as a command-line option. I'll make it private and I don't think deprecation is needed since ``--force`` has always been the correct flag to use.